zbus/src/object_server/interface.rs

```diff
@@ -70,6 +70,11 @@ pub trait Interface: Any + Send + Sync {
     where
         Self: Sized;
 
+    /// Whether methods will be called with a spawned task (and potentially out of order)
```

Comment:
* "with" -> "from".
* I think "out of order" isn't correct. The task are spawned in order, just that they may not be scheduled or the dispatched methods may not respond in order. Perhaps better add a separate note instead:

```
Note: When methods are called from separate tasks, they may not return in the order in which they were called. Typically, this is not a problem and therefore, the default implementation returns `true`.
```

That way, we also document the default value.

Reply:
It's not simply about "return in the order", it's also execution. I disagree about "Typically, this is not a problem" too.